itests/gateway_test.go

```diff
@@ -297,3 +359,209 @@ func sendFunds(ctx context.Context, fromNode *kit.TestFullNode, fromAddr address
 
 	return nil
 }
+
+func TestGatewayRateLimits(t *testing.T) {
+	req := require.New(t)
+
+	kit.QuietMiningLogs()
+	ctx := context.Background()
+	tokensPerSecond := 10
+	requestsPerMinute := 30 // http requests
+	nodes := startNodes(ctx, t,
+		withNodeOpts(kit.DisableEthRPC()),
+		withPerConnectionAPIRateLimit(tokensPerSecond),
+		withPerHostRequestsPerMinute(requestsPerMinute),
+	)
+
+	time.Sleep(time.Second)
```

Comment:
Why? Can we avoid this with `require.Eventually`?

Reply:
we have a `WaitTillChain` that I can use here, just to make sure the chain is ticking